src/main/java/org/json/XMLParserConfiguration.java

```diff
@@ -56,6 +59,11 @@ of this software and associated documentation files (the "Software"), to deal
      */
     private boolean convertNilAttributeToNull;
 
+    /**
+     * This will allow type conversion for values in XML if xsi:type attribute is defined
+     */
+    public Map<String, XMLXsiTypeConverter<?>> xsiTypeMap;
+
     /**
      * Default parser configuration. Does not keep strings (tries to implicitly convert
      * values), and the CDATA Tag Name is "content".
```

Comment:
In the default constructor here, please add a default [emptyMap](https://docs.oracle.com/javase/6/docs/api/java/util/Collections.html#emptyMap())